### ISSUE TITLE
[feature] user activity feed from github

### ISSUE THREAD

**Issue by serapath** [wizardamigos/profiles#12]
to retrieve all recent user activities of the projects a user lists in his `config.json` and display it as a list of links to click and open the user activity on github in a new tab

**Comment by kiecoo** [comment]
get-activity-feed
https://codepen.io/kiecoo/pen/JpPPGg

**Comment by serapath** [comment]
I just wrote to @jameslin3263 about how to make a repository and github page with a little demo for the module he wrote.
Maybe you should try the same. Check: https://github.com/wizardamigos/profiles/issues/10

**Comment by kiecoo** [comment]
OK~I could do it

**Comment by kiecoo** [comment]
【final--can work in codepen】 get-activity-feed
https://codepen.io/kiecoo/pen/jZPPWM?editors=1010

-----------------
@serapath 
I have a problem about make a github page of this module that I failed to solve
(I failed in version 1 so I tried another way in version 2.  version 2 still failed)

**【differences between version 1 &2】**
1. [failed] version_1:  directly move code from [codepen](https://codepen.io/kiecoo/pen/jZPPWM?editors=1010)(which can work successfully in codepen)
(put all code in js)
> html:               <script src="https://unpkg.com/npm-require"></script>
2. [failed] version_2: I try to follow the same way as [ get_userprofiles-from-github_module](https://github.com/kiecoo/get-userprofiles-from-github)   we've done
( separate  code into html & js)
> html:            <!doctype html>
> <html>
>   <head>
>     <meta charset="utf-8">
>     <script src="https://unpkg.com/npm-require"></script>
>     <script> var module = {} </script>
>     <script src="index.js"></script>
>   </head>
>   <body>
>     <script>
>     // var dc = require('dom-console', '5.1.0')()
>     // dc.api.toggle()
>     var bel = require('bel')
>     var csjs = require('csjs-inject')
>     var getGithubData = module.exports


**【list of link】the module of get-activity-feed**
1. [failed] version_1:  [github_page](https://kiecoo.github.io/get-user-activity-feed-github/) & [github_repo](https://github.com/kiecoo/get-user-activity-feed-github)
2. [failed] version_2: [github_page](https://kiecoo.github.io/get-user-activity-feed-github2/)  & [github_repo](https://github.com/kiecoo/get-user-activity-feed-github2)

**Comment by serapath** [comment]
this one shows all the data for me:
* https://kiecoo.github.io/get-user-activity-feed-github2/

this one seems to have an index.html with only a single line of code:
* `<script src="https://unpkg.com/npm-require"></script>` inside the `index.html` which is not enough
*  https://kiecoo.github.io/get-user-activity-feed-github/

**Comment by kiecoo** [comment]
haha, in version2 I found sometimes it works &sometimes it doesn't (maybe it's the reason of over limit as last time)  
so last time I run it &failed.  today I run 2 times(one worked successfullly, the other time failed)

 in this issue I would use version 2 , thank you so much

**Comment by serapath** [comment]
yes, i think we just have to make the github login work and make the logic so, that only AFTER a successful login we display all the data, so that we don't run in or over the API limit :-)
